### PR TITLE
Beastkin Automation

### PR DESCRIPTION
**Pull request by MrVauxs**
This PR adds:
- An ItemAlteration rule to **Critter Form**, adding part of the description to the Change Shape action, describing how you can turn into a pest 1/hour
- A Note for **Quick Shape** on Initiative Rolls
- **Dire Form** Automation
- **Animal Form** Automation* (see below)
- The **Change Shape** effect into the action so you can Use it, instead of having the effect in the description

This PR cannot be accepted in its current state because:
- [x] ~~Does not have i18n strings for some of its choices~~
- [x] ~~Animal Form has two issues in regards to it being based off of 4 battleform spells, it either requires the creation of **~26 duplicate Battleform REs** in order to facilitate all of them, or code in the ability to both **increase the duration of GrantItem'd effects** and **increase their level** (on effect-granting actions).~~ 
**Below issue invalidates this problem by having to modify the Battleform Rule Elements anyway.**
- [x] Lack of support for _"[you] can use your humanoid form's AC if it's higher than what the animal form grants"_ (#15357)

I can chop up this PR so at least some of the changes can be merged already, such as **Dire Form** and **Quick Shape** automation.

### CODE REVIEW

**Comment by pedrogrullada** [comment]
Localization for choices *is* required. You can add them in `re-en.json` under `PF2E.SpecificRule.Beastkin`.

File: packs/feats/critter-shape.json [hunk 1]
```diff
@@ -24,7 +24,28 @@
...
+                "predicate": [
+                    "item:slug:change-shape-beastkin",
+                    {
+                        "not": "feat:animal-shape"
+                    }
+                ],
+                "property": "description",
+                "value": [
+                    {
+                        "text": "Once per hour, you can use @UUID[Compendium.pf2e.actionspf2e.Item.Change Shape (Beastkin)]{Change Shape} to enter a critter shape. While in critter shape, you gain the effect of a 1st-rank @UUID[Compendium.pf2e.spells-srd.Item.Pest Form], except you can only transform into an animal matching your inherent animal."
```
Comment (pedrogrullada) [inline packs/feats/critter-shape.json:41]: Localize this too

File: packs/feats/animal-shape.json [hunk 2]
```diff
@@ -24,7 +24,129 @@
...
+                    "label": "Critter",
+                    "value": "critter"
+                }
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "add",
+                "path": "flags.pf2e.beastkin.forms",
+                "value": {
+                    "label": "Animal",
```
Comment (pedrogrullada) [inline packs/feats/animal-shape.json:145]: We can use the trait for this
```suggestion
                    "label": "PF2E.TraitAnimal",
```

File: packs/feats/animal-shape.json [hunk 3]
```diff
@@ -24,7 +24,129 @@
...
+                ],
+                "flag": "beastkin-animal",
+                "key": "ChoiceSet"
+            },
+            {
+                "actorFlag": true,
+                "adjustName": false,
+                "choices": [
+                    {
+                        "label": "Yes",
```
Comment (pedrogrullada) [inline packs/feats/animal-shape.json]:
```suggestion
                        "label": "PF2E.UI.RuleElements.ChoiceSet.YesLabel",
```

File: packs/feats/animal-shape.json [hunk 4]
```diff
@@ -24,7 +24,129 @@
...
+                        "value": 1
+                    },
+                    {
+                        "label": "No",
```
Comment (pedrogrullada) [inline packs/feats/animal-shape.json]:
```suggestion
                        "label": "PF2E.UI.RuleElements.ChoiceSet.NoLabel",
```

File: packs/feats/animal-shape.json [hunk 5]
```diff
@@ -24,7 +24,129 @@
...
+                "adjustName": false,
+                "choices": [
+                    {
+                        "label": "PF2E.UI.ChoiceSet.YesLabel",
+                        "value": 1
+                    },
+                    {
+                        "label": "PF2E.UI.ChoiceSet.NoLabel",
+                        "value": 0
+                    }
```
Comment (pedrogrullada) [inline packs/feats/animal-shape.json:122]: Missed `RuleElements` in between there
```suggestion
                    {
                        "label": "PF2E.UI.RuleElements.ChoiceSet.YesLabel",
                        "value": 1
                    },
                    {
                        "label": "PF2E.UI.RuleElements.ChoiceSet.NoLabel",
                        "value": 0
                    }
```

**Comment by MrVauxs** [comment]
Superseded by #16267. I may pick up on this later if further automation is needed, most of the stuff here can be still used.

**Pull request closed without merging**